Kenneth-Macharia/Store-Manager-api
# #161260057 Get all products tdd

What does this PR do?
-Adds test

Description of Task to be completed?
-Adds passing tests for the POST/products endpoint, functional.
-Add TDD for the GET/ products endpoint, not yet functional.

How should this be manually tested?
Clone this repo and cd into it.
-Activate the virtual environment 'env' at the prompt.
-Install dependancies: 'pip install -r requirements.txt
-Run pytest at the prompt: 'py.test'
-Test results should as shown in the screeshot below:

![screen shot 2018-10-16 at 11 45 51 pm](https://user-images.githubusercontent.com/42747960/47046432-be71cd80-d19d-11e8-8a09-ca289ced241b.png)

What are the relevant pivotal tracker stories?
#161260057
